package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@contentful/live-preview",
-  "version": "1.0.1",
```

Comment:
Github is not automatically doing this, must be something wrong with the setup 🤔

Reply:
mhh that strange 🤔

Reply:
we need this plugin for updating the package.json version:
https://github.com/semantic-release/git

Reply:
I think this is already in the package.json